tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -155,32 +178,14 @@ module.exports = {
     changeCustomPermissionsTo: async function (collaborator, requiredPermissions) {
       const requiredPermissionArray = this.getArrayFromPermissionString(requiredPermissions)
 
-      let permission = ''
-      this.expandInformationSelector(collaborator)
-
+      await this.expandInformationSelector(collaborator)
```

Comment:
This does not look like an `async` function. But, it should be good. If `await` is used before a value, it gets converted to a resolved promise, and waits for it. 

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#Examples